### PR TITLE
Mc user management

### PR DESCRIPTION
**Pull request by FBRTMaka**
@oceanzus , @lukecampbell 

A bit to insert the user scopes to be checked on by the UserScopeLink class

### CODE REVIEW

File: ooiservices/manage.py [hunk 1]
```diff
@@ -73,11 +73,11 @@ def test(coverage=False):
 
 @manager.option('-p', '--password', required=True)
 def deploy(password):
-
     from flask.ext.migrate import upgrade
     from ooiservices.app.models import User, UserScope, UserScopeLink, Array
     from ooiservices.app.models import PlatformDeployment, InstrumentDeployment, Stream, StreamParameterLink
     #Create the local database
+    app.logger.info('Creating DEV and TEST Databases')
```
Comment (lukecampbell) [inline ooiservices/manage.py]: Thank you for using log

**Comment by lukecampbell** [comment]
If you could make those changes to use `sh` then I'll merge.

**Comment by FBRTMaka** [comment]
you got a example that uses -U postgres and the db target?

**Comment by lukecampbell** [comment]
I'm just looking to avoid os.system, it's plagued with insecurities.

**Comment by lukecampbell** [comment]
```python
from sh import psql
psql(target, '-c', command, '-U', user)
```

**Pull request merged**